src/helpers/objects.processor.js

```diff
@@ -99,7 +99,7 @@ function dictionaryGetter (str) {
 function getDictionaryObject (str) {
     const dictionaryGetterFunc = stepsConfig.objectsProcessor.dictionaryGetter || dictionaryGetter;
     const value = dictionaryGetterFunc(str);
-    const idValue = value.replace(_r(regDynamicId, 'g'), stepsConfig.id.getId());
+    const idValue = value.replace(_r(regDynamicId, 'g'), stepsConfig.id.id);
```

Comment:
`stepsConfig.id.id` - sounds a bit strange - maybe it will be better to name it `stepsConfig.id.idValue`?